monkey/infection_monkey/utils/commands.py

```diff
@@ -0,0 +1,61 @@
+import logging
+
+from infection_monkey.model import CMD_CARRY_OUT, CMD_EXE, MONKEY_ARG
+
+LOG = logging.getLogger(__name__)
+
+
+def build_monkey_commandline(target_host, depth, vulnerable_port, location=None):
+    from infection_monkey.config import GUID
```

Comment:
Can we move this to other imports? If not, why is this here? Maybe we can avoid circular imports somehow?